src/services/accounts/AccountsStakingPayoutsService.ts

```diff
@@ -466,21 +467,15 @@ export class AccountsStakingPayoutsService extends AbstractService {
 		address: string,
 		deriveEraExposure: DeriveEraExposure
 	): DeriveEraExposureNominating[] | undefined {
-		let nominatedExposures: DeriveEraExposureNominating[] | undefined =
-			deriveEraExposure.nominators[address];
+		let nominatedExposures: DeriveEraExposureNominating[] =
+			deriveEraExposure.nominators[address] ?? [];
```

Comment:
dq: why is `??` better here than plain old `||`?

Reply:
No amazing reason, just figured it was more descriptive because it shows we just expect undefined/null.